packages/tpetra/core/src/Tpetra_BlockCrsMatrix_def.hpp

```diff
@@ -1111,6 +1111,13 @@ class GetLocalDiagCopy {
     const Scalar one_minus_omega = Teuchos::ScalarTraits<Scalar>::one()-omega;
     const Scalar     minus_omega = -omega;
 
+    // FENCE REVIEW - CONFIRMED FAILURE
+    //   Testing: This code is exercised by unit tests.
+    //   White:   Fails Ifpack2Relaxation_double_int_longlong_TestLowerTriangularBlockCrsMatrix_UnitTest
+    //            Ifpack2Relaxation_double_int_longlong_TestUpperTriangularBlockCrsMatrix_UnitTest
+    //   Plan:
+    //   Notes:
+
```

Comment:
@brian-kelley  Will this code (function localGaussSeidel) be going away with your refactor of Ifpack2 and Tpetra?  If so, we can leave the fence here until you do your refactor, which I assume will remove localGaussSeidel from Tpetra.   If not, I will look more carefully at the fence.

Reply:
@kddevin Yes, that will be going away.